web/html/src/manager/maintenance/calendar/web-calendar.tsx

```diff
@@ -49,10 +50,11 @@ const WebCalendar = (props: WebCalendarProps) => {
   const getEvents = (operation) => {
     const date = getDate(operation);
     if (operation === "initial" || needsUpdate(date)) {
-      const endpoint = `/rhn/manager/api/maintenance/events/${operation}/${props.type}/${date.valueOf()}/${props.id}`;
+      const startOfWeek = getApi().currentDataManager.data.dateEnv.weekDow;
```

Comment:
I couldn't find any reference to `weekDow` in the full calendar api docs. Is this safe to use? What about https://fullcalendar.io/docs/firstDay ?

Reply:
Moreover: is this passing of the first day necessary? Wouldn't it be enough to compute the first day of week based on the users preferred locale on the java side?

Reply:
That one let's you set the first day of the week that `fullcalendar` should use, but it actually has built in support for the default start of the week depending on locale.

The second one is a bit of a difficult one. Let me try to explain.
The java class that spits out the first day of the week expects locales passed to it to have both language and country defined (meaning e.g. de_DE) but for a lot of the locales we use in suma we do not define the country (e.g. de, fr) and just passing the language (e.g. de) will always result in sunday being the first day of the week.

Since I didn't want to go through guessing the country we are using for these cases I decided to take `fullcalendars` build in support for this as the source of truth.
I actually went through all the languages we use and compared `fullcalendars` built in default with what's supposed to be the default for the country using the language and they matched in all the cases. So I think we are on the safe side in this case.

Reply:
Ok, thanks for explaining.